rspec-mode.el

```diff
@@ -225,6 +225,13 @@ info, are considered errors."
   :type 'boolean
   :group 'rspec-mode)
 
+(defcustom rspec-holder-dirs '("app" "lib")
```

Comment:
I've spent some time thinking about the name. It needs to mean directories that contain source files, but whose names have no meaning, so to speak, so they're omitted from the test file names.

"holder dirs" is not ideal, but "code dirs" seems more vague to me. To take the example from issue 123, it's just as valid to call `services` a "code dir", yet we'll want it to stay out of this list.